synapse/rest/client/room.py

```diff
@@ -60,6 +62,30 @@
 
 logger = logging.getLogger(__name__)
 
+messsages_response_timer = Histogram(
```

Comment:
Can you add a comment to say why we're adding this on top of the existing metrics please?